volumes/vds/readme.md

```diff
@@ -0,0 +1,448 @@
+# Vehicle Detector System (VDS) data  
+
+# Table of contents
+1. [Introduction](#introduction)
+    - [Summary of Improvements](#summary-of-improvements)
+    - [Overview of Sensor Classes](#overview-of-sensor-classes)
+    - [Data Availability](#data-availability)
+    - [Future Work](#future-work)
+2. [Table Structure](#table-structure)
+    - [vds.raw_vdsdata](#vdsraw_vdsdata)
+    - [vds.counts_15min](#vdscounts_15min)
+    - [vds.counts_15min_bylane](#vdscounts_15min_bylane)
+    - [vds.raw_vdsvehicledata](#vdsraw_vdsvehicledata)
+    - [vds.veh_speeds_15min](#vdsveh_speeds_15min)
+    - [vds.veh_length_15min](#vdsveh_length_15min)
+    - [vds.vdsconfig](#vdsvdsconfig)
+    - [vds.entity_locations](#vdsentity_locations)
+    - [vds.detector_inventory](#vdsdetector_inventory)
+    - [vds.detector_inventory_cursed](#vdsdetector_inventory_cursed)
```

Comment:
I think there are enough tables that sub-headings might be warranted. 

- Look up tables and views
- Aggregate Tables
- Raw Data
- Cursed